src/select/multi-value.js

```diff
@@ -19,10 +19,7 @@ export default function MultiValue(props: any) {
       overrides={{
         Root: {
           style: ({$theme: {sizing}}) => ({
-            marginRight: sizing.scale0,
-            marginBottom: 0,
-            marginLeft: sizing.scale0,
-            marginTop: 0,
+            margin: sizing.scale0,
```

Comment:
can you please write out all directions? this is so that we have proper layout with styletron's atomic classes: https://www.styletron.org/concepts/#shorthand-and-longhand-properties

when giving examples we like to be more verbose, someone referencing this code may choose to use shorthand property names if they wish

```suggestion
            marginTop: sizing.scale0,
            marginRight: sizing.scale0,
            marginBottom: sizing.scale0,
            marginLeft: sizing.scale0,            
```

Reply:
Sure
updated

Reply:
thanks!